tests/std/tests/P0896R4_views_drop_while/test.cpp

```diff
@@ -0,0 +1,431 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <algorithm>
+#include <cassert>
+#include <forward_list>
+#include <ranges>
+#include <span>
+#include <type_traits>
+#include <utility>
+#include <vector>
+
+#include <range_algorithm_support.hpp>
+using namespace std;
+
+// Test a silly precomposed range adaptor pipeline
+template <int X>
+constexpr auto is_less_than = [](const auto& x) { return x < X; };
+
+using Pred = remove_const_t<decltype(is_less_than<3>)>;
+STATIC_ASSERT(is_nothrow_copy_constructible_v<Pred>&& is_nothrow_move_constructible_v<Pred>);
+
+constexpr auto pipeline = views::drop_while(is_less_than<3>) | views::drop_while(is_less_than<3>)
+                          | views::drop_while(is_less_than<3>) | views::drop_while(is_less_than<3>);
+
+template <class Rng, class V = views::all_t<Rng>>
+using pipeline_t = ranges::drop_while_view<
+    ranges::drop_while_view<ranges::drop_while_view<ranges::drop_while_view<V, Pred>, Pred>, Pred>, Pred>;
+
+template <class Rng>
+concept CanViewDrop_while = requires(Rng&& r) {

```

Comment:
Nitpick: Would `CanViewDropWhile` be better?

Reply:
Ugh, I caught the abominable `CanViewTake_while`, but slid right by this one.

Reply:
Urgh, i had a look yesterday but it was late thanks for finding these